Example/webpack.config.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
         port: 4001,
         disableHostCheck: true,
         historyApiFallback: true,
+        headers: {
+            'Access-Control-Allow-Origin': '*',
```

Comment:
@sertony do we want it to be accessible for everyone? 🤔

Reply:
Hm, I believe that we should set it in firebase.json file when deploying the website into its hosting as we do for Surf:
https://github.com/tonlabs/TON-Surf/blob/development/web/firebase.json

More details can be found here: https://firebase.google.com/docs/hosting/full-config

@savelichalex I guess you'll need to help setting it up!